<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Motion_control</class>
 <widget class="QWidget" name="Motion_control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>272</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transform</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QPushButton" name="pb_alignment">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>AlineOri</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pb_setas_benchmark">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>SetAsOri</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pb_reset_benchmark">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>ResetOri</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="cb_reverse">
       <property name="text">
        <string>reversal</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pb_resetMove">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>ResetMove</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pb_movetoOri">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>MoveToOri</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pb_reverse">
       <property name="text">
        <string>Reverse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="0">
      <widget class="QLabel" name="label_movestp">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px</string>
       </property>
       <property name="text">
        <string>move step(mm)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="sb_movestp">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>200.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>20.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="sb_rotstp">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>5.000000000000000</double>
       </property>
       <property name="maximum">
        <double>175.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>5.000000000000000</double>
       </property>
       <property name="value">
        <double>45.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pb_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_rotstp">
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:0px</string>
       </property>
       <property name="text">
        <string>rot step(mm)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pb_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_moveXPos">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>X+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_moveZPos">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Z+</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_moveYNeg">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Y-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_moveYPos">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Y+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn_moveXNeg">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>X-</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btn_moveZNeg">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Z-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pb_rotX">
       <property name="text">
        <string>Rot X</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pb_rotY">
       <property name="text">
        <string>Rot Y</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pb_rotZ">
       <property name="text">
        <string>Rot Z</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
